backend/.pre-commit-config.yaml

```
@@ -0,0 +1,87 @@
+# Pre-commit hooks configuration for SEPE Templates Comparator
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)